.github/workflows/rust.yml

```diff
@@ -6,6 +6,9 @@ on:
       - main
   pull_request:
 
+env:
+  RUSTFLAGS: -Coverflow-checks=y
```

Comment:
sorry but last nit :D I'm wondering if this makes `cargo check` and `cargo clippy` slowlier since this env will be applied to all of them instead of just the the tests. Perhaps `RUSTFLAGS="-Coverflow-checks=y" cargo test --release` is better?

Reply:
Ideally, they share the build cache, so using the same flags should save the overall time. (But the incremental build is not enabled by default in recent rust releases)

Reply:
I just compared the build logs, the performance impact of build time is negligible, I guess it's OK but plz let me know if u still like applying the flags to cargo test only.

```
Finished release [optimized] target(s) in 1m 31s
```

```
Finished release [optimized] target(s) in 1m 34s
```

Reply:
To further speed up the job, I commented out `cargo test --release` because it's covered by `cargo test --release --features ocaml_types`